src/common/query/src/prelude.rs

```diff
@@ -25,3 +25,5 @@ pub const GREPTIME_TIMESTAMP: &str = "greptime_timestamp";
 pub const GREPTIME_VALUE: &str = "greptime_value";
 /// Default counter column name for OTLP metrics.
 pub const GREPTIME_COUNT: &str = "greptime_count";
+/// Default physical table name
+pub const GREPTIME_PHYSICAL_TABLE: &str = "greptime_physical_table";
```

Comment:
What about using a query parameter instead of an HTTP header?

```yaml
remote_write:
  - url: "http://localhost:4000/v1/prometheus/write?physical_table=my_physical_table"
```

It doesn't feel right as an HTTP header; it's actually a request parameter.